explorer/site-config/hca-dcp/dev/projectViewModelBuilder.ts

```diff
@@ -130,6 +133,20 @@ export const buildHero = (
   };
 };
 
+/**
+ * Build props for project index species NTagCell component from the given projects response.
+ * @param projectsResponse - Response model return from projects API.
+ * @returns model to be used as props for the project index species NTagCell.
+ */
+export const buildIndexSpecies = (
```

Comment:
What about moving index-specific functionality to a different builder, e.g. `projectsViewModelBuilder`?